features/README.md

```diff
@@ -1,35 +1,22 @@
-# Running Deadalus acceptance tests
+# Running Daedalus acceptance tests
 
 
-1. Make sure you have correct node/npm versions installed on your machine (node v6.x and npm v3.x)
-2. Clone Deadalus repo to your machine (git@github.com:input-output-hk/daedalus.git - use **master** branch)
+1. Make sure you have the correct node/npm versions installed on your machine (node v8.x and npm v5.x)
+2. Clone Daedalus repository to your machine (`git clone git@github.com:input-output-hk/daedalus.git` - use **master** branch)
 3. Install npm dependencies from within Daedalus directory:
-```
+
+```bash
 $ cd daedalus/
 $ npm install
 ```
-4. Link your backend build with the Daedalus frontend (assumed that you have build the backend previously):
-```
-$ cd daedalus/node_modules/
-$ npm link daedalus-client-api
-```
-5. Launch the backend (cardano-sl) in production/staging mode:
-```
-$ cd cardano-sl/
-$ ./scripts/launch/staging.sh
-```
-6. Run Deadalus frontend in hot-server mode:
-```
-$ cd daedalus/
-$ npm run hot-server
-```
-7. Run Daedalus frontend tests in a separate Terminal window (leaving hot-server to run in the other):
-```
+4. Build and run the backend (Cardano SL) following the instructions from [Daedalus](https://github.com/input-output-hk/daedalus/blob/master/README.md#development---with-cardano-wallet) README file.
+5. Run Daedalus frontend tests:
+
+```bash
 $ cd daedalus/
 $ npm run test
 ```
 
-There is a total of 46 different acceptance tests.
-Whole test suite takes around 5 minutes to finish.
+There is a total of 60 different acceptance tests.  
```

Comment:
Not sure we should put concrete numbers here as this will often change in the future.

Reply:
It's more of a show off thing I guess :)
But to be honest our total test cases count has not been changing that much...